spec/factories/work_activity.rb

```diff
@@ -5,6 +5,6 @@
     message { "test work activity message" }
     activity_type { "SYSTEM" }
     created_by_user_id { FactoryBot.create(:user).id }
-    work_id { FactoryBot.create(:work).id }
+    work_id { FactoryBot.create(:draft_work).id }
```

Comment:
Not a problem, but also not sure what motivates this particular change...

Reply:
You are right this is an unrelated change.  This factory was not utilized an errors when you do. I'll update the PR comment to highlight that this is also changed.